starknet_py/net/rpc_schemas/rpc_schemas.py

```diff
@@ -170,6 +170,7 @@ class StarknetBlockSchema(Schema):
         data_key="transactions",
     )
     timestamp = fields.Integer(data_key="timestamp")
+    starknet_version = fields.String(data_key="starknet_version")
```

Comment:
I don't think the version of the RPC we support (0.15) has this fields. I think it'd be better to have it with `load_default` here.